tests/security/cc/crypto.pm

```diff
@@ -0,0 +1,37 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+#
+# Summary: Run 'crypto' test case of 'audit-test' test suite
+# Maintainer: llzhao <llzhao@suse.com>
+# Tags: poo#95485
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use audit_test qw(run_testcase compare_run_log);
+
+sub run {
+    my ($self) = shift;
+
+    select_console 'root-console';
+
+    # Export MODE
+    assert_script_run("export MODE=$audit_test::mode");
```

Comment:
`export MODE=xxx` isn't needed here.

Reply:
There is a reboot needed after SELinux enabled so the  `export MODE=xxx` is needed here.